torchvision/prototype/transforms/_auto_augment.py

```diff
@@ -72,11 +72,10 @@ def _apply_image_transform(
 
         # Fill = 0 is not equivalent to None, https://github.com/pytorch/vision/issues/6517
         # So, we have to put fill as None if fill == 0
-        fill_: Optional[Union[int, float, Sequence[int], Sequence[float]]]
+        # This is due to BC with stable API which has fill = None by default
+        fill_ = F._geometry._convert_fill_arg(fill)
```

Comment:
We might want to consider making `_convert_fill_arg()` public on the future as this is regularly used in Transforms as a utility method.